test/extended/testdata/builds/volumes/docker-buildconfig.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
+kind: List
```

Comment:
non-blocking nit - why a List instead of just the BuildConfig YAML?

Reply:
Leftover from the yaml I copied it from that did not get cleaned up.

Reply:
updated in next push